source/lexbor/html/serialize.c

```diff
@@ -317,7 +313,7 @@ lxb_html_serialize_node_cb(lxb_dom_node_t *node,
     return LXB_STATUS_OK;
 }
 
-lxb_inline lxb_status_t
+bool
```

Comment:
Let's make it static.

```C
static bool
lxb_html_serialize_node_is_void(lxb_dom_node_t *node);
```
```C
static bool
lxb_html_serialize_node_is_void(lxb_dom_node_t *node)
{
    /* ... */
}
```

Reply:
@kostya 

If this function is needed externally, then you need to rename it to `lxb_html_node_is_void` and move it to `source/lexbor/html/html.c`.

Reply:
And for this you need a separate patch. No need to mix different edits in one patch.